runtime/src/main/java/org/corfudb/runtime/FastSmrMapsLoader.java

```diff
@@ -0,0 +1,323 @@
+package org.corfudb.runtime;
```

Comment:
could we make this part of something like package "org.corfudb.runtime.recovery"?